java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -113,33 +114,53 @@ public void setUp() throws IOException, InterruptedException {
   /**
    * Prepare bazel for running, and return the {@link Command} object to run it.
    */
-  public Command bazel(String... args) throws IOException {
-    return bazel(new ArrayList<>(Arrays.asList(args)));
+  public Command bazel(String... args) {
```

Comment:
I think the area of setting up the bazel command is starting to become complex (7 methods already)
@anchlovi have you decided against the builder we talked about?

Reply:
:) also thought to say builder but didn’t want to complicate his efforts even more.
OTOH now is the best time to break APIs as we learn them. If we decide on it then let’s loop in Damien as the other main user of this lib

Reply:
I think that maybe it's better to defer the builder change to another PR since we should probably also discuss it with Damien. WDYT?